drivers/note/note_driver.c

```diff
@@ -259,7 +261,17 @@ static void note_common(FAR struct tcb_s *tcb,
 #else
   /* Save the LS 32-bits of the system timer in little endian order */
 
-  sched_note_flatten(note->nc_systime, &systime, sizeof(systime));
+#ifdef CONFIG_SCHED_INSTRUMENTATION_PERFCOUNT
+  /* Use perf count as tv_nsec to achieve high -precision timestamp */
+
+  up_perf_convert(count, &ts);
+#  else
+  ts.tv_nsec = (systime % (1000 * 1000 / CONFIG_USEC_PER_TICK))
```

Comment:
Let's remove systime? which is no difference from clock_systime_timespec. So change CONFIG_SCHED_INSTRUMENTATION_HIRES to CONFIG_SCHED_INSTRUMENTATION_PERFCOUNT and remove systime.